From 63357d5b4ee611ff94e451dd3971ec22df7f822f Mon Sep 17 00:00:00 2001
From: Ghennadi Procopciuc <ghennadi.procopciuc@nxp.com>
Date: Thu, 12 Jan 2023 10:25:09 +0200
Subject: [PATCH 27/63] arch/arm64: Enable mmio-mux for S32CC platforms

mmio-mux is used by the S32CC platforms to control CAN
timestamping.

Issue: ALB-9261
Upstream-Status: Pending 

Signed-off-by: Ghennadi Procopciuc <ghennadi.procopciuc@nxp.com>
Signed-off-by: Zhantao Tang <zhantao.tang@windriver.com>
---
 arch/arm64/Kconfig.platforms | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/arch/arm64/Kconfig.platforms b/arch/arm64/Kconfig.platforms
index f8bbc92325fb..fd703e3be86a 100644
--- a/arch/arm64/Kconfig.platforms
+++ b/arch/arm64/Kconfig.platforms
@@ -19,6 +19,8 @@ choice
 		bool "S32 GEN1 SOC"
 		select NXP_ERRATUM_ERR050543
 		select MAILBOX
+		select MULTIPLEXER
+		select MUX_MMIO
 		select ARM_SMC_MBOX
 		select ARM_SCMI_PROTOCOL
 		select COMMON_CLK_SCMI
-- 
2.25.1

